<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formulario</class>
 <widget class="QWidget" name="formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>340</width>
     <height>372</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Usuario:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_nombre_usu">
        <property name="styleSheet">
         <string notr="true">color:rgb(204, 0, 0)</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Edad:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_edad_usuar">
        <property name="styleSheet">
         <string notr="true">color: rgb(204, 0, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Genero:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="Genero_r"/>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="nombre_f"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Direccion:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="doc_f"/>
        </item>
        <item row="8" column="0">
         <widget class="QLineEdit" name="direccion_f"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="apellido_f"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Raza:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QComboBox" name="Raza_r"/>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Fecha de nacimiento:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="fecha_f"/>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Nivel de ingreso:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Documento de identidad:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="nivel_f">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">color: rgb(52, 101, 164);</string>
          </property>
          <property name="text">
           <string>REGISTRO PACIENTE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Apellido:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Duracion del examen</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="_time30">
             <property name="text">
              <string>30 Segundos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_time1m">
             <property name="text">
              <string>1 minutos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="_time2m">
             <property name="text">
              <string>2 minutos</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="iniciar_f">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Iniciar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="salir_f">
            <property name="text">
             <string>Salir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
